packages/common/src/stores/AppController.ts

```diff
@@ -57,16 +62,17 @@ export default class AppController {
 
     useConfigStore.setState({ config, accessModel, integrationType, clientId, isSandbox, offers, loaded: true });
 
-    this.configService.maybeInjectAnalyticsLibrary(config);
-
     return config;
   };
 
-  initializeApp = async () => {
+  initializeApp = async (url: string) => {
     const settings = await this.settingsService.initialize();
-    const configSource = this.settingsService.getConfigSource(settings);
+    const configSource = await this.settingsService.getConfigSource(settings, url);
     const config = await this.loadAndValidateConfig(configSource);
 
+    // we can add the configSource to the storage prefix, but this will cause a breaking change for end users
```

Comment:
Ah, you mention it here. Don't we want to use the `configId` as a key partial anymore?

Reply:
I initially thought of moving the `window.configId` here, but because `window.configId` is not being used, we introduce a breaking change.

Reply:
Okay, acceptable. But should we add the config id anyway?

Reply:
Nope, by using `configSource` here, which is always set, we would invalidate the personal shelves for all users and `window.configId` can't be used anymore.

Reply:
I understand you now. I suggest changing the comment to something like such:
`// we could add the configSource to the storage prefix, but this would cause a breaking change for end users (since 'window.configId' isn't used anymore, all platforms use the same prefix)`